docs/restic.md

```diff
@@ -25,6 +25,7 @@ cross-volume-type data migrations. Stay tuned as this evolves!
 
 - A working install of Ark version 0.10.0 or later. See [Set up Ark][2]
 - A local clone of [the latest release tag of the Ark repository][3]
+- Enable Mount propagation feature on Kubernetes Cluster. Requires Kubernetes v1.10 or higher. See [Setup MountFlags][6]
```

Comment:
The `MountPropagation` feature is beta in Kubernetes 1.10 and enabled by default. Would it make more sense to say that Ark's restic integration requires Kubernetes 1.10 or higher (make sure you have not disabled the `MountPropagation` feature gate)? We could then also update our example yaml files for the restic DaemonSets to configure `mountPropagation`. I'm guessing we need `hostToContainer`?

Reply:
Thanks @ncdc - Modified the text - let me know if this sounds better. 

I have currently tested this with gcp without making any change to the example yaml. Once I have tested on other platforms, I will raise a new PR with updated example yamls.